pkg/kubectl/cmd/util/factory.go

```diff
@@ -554,10 +557,6 @@ func recordCommand(args []string) string {
 	return strings.Join(args, " ")
```

Comment:
can you fix this to copy the passed in args (not mutate them), and use `filepath.Base(args[0])` instead of hard-coding `kubectl`?